packages/api/src/external/fhir/document/index.ts

```diff
@@ -247,19 +271,122 @@ function convertCWNameToHumanName(
 /**
  * Converts a CW HumanName to a FHIR HumanName
  *
- * @param n
+ * @param name
  * @returns
  */
-function getHumanNamesFromObject(n: CWHumanName): HumanName[] {
+function getHumanNamesFromObject(name: CWHumanName): HumanName[] {
   const names: HumanName[] = [];
-  if (n.family.length) {
-    for (let i = 0; i < n.family.length; i++) {
-      names.push({
-        family: n.family[i],
-        given: n.given && n.given[i] ? [n.given[i]] : undefined,
-        prefix: n.prefix && n.prefix[i] ? [n.prefix[i]] : undefined,
-      });
-    }
-  }
+  if (name.use === "unspecified") return names;
+
+  const namePrefix = getHumanNameAttribute(name.prefix);
+  const nameSuffix = getHumanNameAttribute(name.suffix);
+
+  names.push({
+    family: joinName(name.family),
+    given: name.given,
+    prefix: namePrefix,
+    suffix: nameSuffix,
+    text: name.text ?? undefined,
+  });
   return names;
 }
+
+/**
+ * Converts a CW Document Identifier to a FHIR Identifier
+ *
+ * @param identifier
+ * @returns
```

Comment:
nit:

- `.` at the end of the sentence
- `@param identifier` should describe what the param is
- `@returns` should describe what the func returns

ditto to all other function docs added in this PR